src/test-helpers.tsx

```diff
@@ -967,6 +971,10 @@ export const createFakeCommentsResponse = (
   return createFakeApiPage<GetCommentsResponse>({ results });
 };
 
+export const createErrorResponse = (errorResponse: { error: Error }) => {
```

Comment:
It would be more convenient if this helper returned a default response with a pre-filled error. This would be helpful in cases where the test does not need to make assertions about the actual error. For example:

```diff
diff --git a/src/test-helpers.tsx b/src/test-helpers.tsx
index 17ce432..def1f93 100644
--- a/src/test-helpers.tsx
+++ b/src/test-helpers.tsx
@@ -971,7 +971,11 @@ export const createFakeCommentsResponse = (
   return createFakeApiPage<GetCommentsResponse>({ results });
 };
 
-export const createErrorResponse = (errorResponse: { error: Error }) => {
+export const createErrorResponse = (
+  errorResponse = {
+    error: new Error('Example error'),
+  },
+) => {
   return errorResponse as ErrorResponseType;
 };
 
```

Reply:
Fixed, added default value for parameter.